nelson-feng/wk02/3-wed/cat.js

```diff
@@ -0,0 +1,107 @@
+// # Exercises: Animation
+
+// ## The Cat Walk
+
+const cat = document.getElementById("black-cat");
+
+cat.style.position = 'absolute';
+cat.style.left = '0px';
+
+const catWalkingRight = function () {
+    cat.style.transform = 'scaleX(1)';
+    const startPosition = parseInt(cat.style.left);
+    const newPosition = startPosition + 1;
+    cat.style.left = newPosition + 'px';
+    if(cat.style.left === '1180px'){
+        clearInterval(intervalId);
+        setInterval(catWalkingLeft, 10);
+    }
+}
+const catWalkingLeft = function () {
+    cat.style.transform = 'scaleX(-1)';
+    const startPosition = parseInt(cat.style.left);
+    const newPosition = startPosition - 1;
+    cat.style.left = newPosition + 'px';
+    if(cat.style.left === '0px'){
+        setInterval(catWalkingRight, 10);
+    }
```

Comment:
You've already managed to pull off turning the cat around. Double-check your code on **lines 15-18** and see if this is a correct mirror of what you've done there.